modules/ROOT/pages/jquery.adoc

```diff
@@ -0,0 +1,125 @@
+= TinyMCE jQuery integration quick start guide
+
+:title_nav: jQuery
+:description: Documentation for the official TinyMCE jQuery integration.
+:keywords: integration integrate jquery javascript
+:productSource: cloud
+
+The https://github.com/tinymce/tinymce-jquery[Official {productname} jQuery component] integrates {productname} into jQuery projects.
+
+
+NOTE: We are pleased to provide integrations/code guidance to help you build great products with {productname}. If you have queries about this integration, post your question on Stack Overflow using the link:{communitysupporturl}[`{prodnamecode}` tag].
+
+== Procedure
+This procedure creates a basic jQuery integration containing a {productname} editor based on our xref:basic-example.adoc[Basic example].
+
+. In a HTML file, add a source script for source jQuery.
++
+If the project loads jQuery from https://code.jquery.com/, use the script provided by the https://code.jquery.com/[jQuery CDN], which includes the `integrity` and `crossorigin` attributes.
+
+. Source {productname} and the {productname} jQuery integration from the {cloudname} or from a self-hosted location.
+
+* *{cloudname}*
++
+To source {productname} from the {cloudname} and the {productname} jQuery integration from a CDN, add the following `script` elements:
++
+[source,html]
```

Comment:
```suggestion
[source,html,subs="attribubtes+"]
```
This is needed to make the `cdnurl` attribute substitution work below.

Reply:
Thanks, I was wondering if that would work.